public/romania/manifest.json

```
@@ -7,50 +7,68 @@
   "theme_color": "#b80000",
   "icons": [
     {
-      "src": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/romania/images/icons/icon-72x72.png?v=1",
+      "src": "/romania/images/icon-72x72.png",
```

We will have to bring back the first part of this src url for this entry and the others in the file, with the updated paths to reflect the missing folders I have commented about above
```suggestion
      "src": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/romania/images/icons/icon-72x72.png",
```